docs/dev/table/sourceSinks.md

```diff
@@ -316,7 +316,7 @@ A table schema field of type `SQL_TIMESTAMP` can be declared as rowtime attribut
 
 * the name of the field, 
 * a `TimestampExtractor` that computes the actual value for the attribute (usually from one or more other attributes), and
-* a `WatermarkStrategy` that specifies how watermarks are generated for the the rowtime attribute.
+* a `WatermarkStrategy` that specifies how watermarks are generated for the the rowtime attribute (With the default value `null` to indicate no watermark should be generated).
```

Comment:
I think we should not use `null` to indicate that no watermarks should be assigned. 
Rather provide a `WatermarkStrategy` that indicates that existing watermarks are preserved like:

```
class PreserveWatermarks extends WatermarkStrategy {
  def INSTANCE: PreserveWatermarks = new PreserveWatermarks
}
```

Reply:
I would leave this part of the docs unchanged